packages/webapp/pages/squads/[handle].tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  GetStaticPathsResult,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+} from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import React, { ReactElement, useContext, useMemo } from 'react';
+import { useRouter } from 'next/router';
+import { NextSeoProps } from 'next-seo/lib/types';
+import { NextSeo } from 'next-seo';
+import Feed from '@dailydotdev/shared/src/components/Feed';
+import { SOURCE_FEED_QUERY } from '@dailydotdev/shared/src/graphql/feed';
+import AuthContext from '@dailydotdev/shared/src/contexts/AuthContext';
+import { SquadPageHeader } from '@dailydotdev/shared/src/components/squads/SquadPageHeader';
+import { FeedPage } from '@dailydotdev/shared/src/components/utilities';
+import {
+  getSquad,
+  getSquadMembers,
+  Squad,
+  SquadMember,
+} from '@dailydotdev/shared/src/graphql/squads';
+import Custom404 from '../404';
+import { defaultOpenGraph, defaultSeo } from '../../next-seo';
+import { mainFeedLayoutProps } from '../../components/layouts/MainFeedPage';
+import { getLayout } from '../../components/layouts/FeedLayout';
+
+type SourcePageProps = {
+  squad: Squad;
+  squadMembers: SquadMember[];
+  squadMemberCount: number;
+};
+
+const SquadPage = ({
+  squad,
+  squadMembers,
+  squadMemberCount,
+}: SourcePageProps): ReactElement => {
+  const { isFallback } = useRouter();
+  const { user } = useContext(AuthContext);
+  // Must be memoized to prevent refreshing the feed
+  const queryVariables = useMemo(
+    () => ({ source: squad?.id, ranking: 'TIME' }),
+    [squad?.id],
+  );
+
+  if (!isFallback && !squad) {
+    return <Custom404 />;
+  }
+
+  if (isFallback || !squad) {
+    return <></>;
+  }
+
+  const seo: NextSeoProps = {
+    title: `${squad.name} posts on daily.dev`,
+    openGraph: { ...defaultOpenGraph },
+    ...defaultSeo,
+  };
+
+  return (
+    <FeedPage
```

Comment:
We need to find a way to make this a ProtectedPage I think.

Reply:
Done